lua/swamp/misc/sh_titles.lua

```diff
@@ -160,6 +160,69 @@ AddTitle("kleinertp", {
     {1, "Test Subject", 10000}
 }, "Be subjected to one of Dr. Isaac Kleiner's teleportation experiments", "s_kleinertp")
 
+AddTitle("buttonsearch", {
+    { 1, "Button Rookie", 0},
+    { 10, "Button Hunter", 0},
+    { 100, "Button Master", 0}
+}, "Find and press %s buttons.", "s_magicbutton")
+
+AddTitle("snowballhit", {
+    {100, "Frosty The Snowman"}
+}, "Hit %s active players with a snowball.", "s_snowballhit")
+
+AddTitle("drinkenergy", {
+    {100, "Boomer"},
```

Comment:
this should be like something where you have to drink monster and then stand around until the effect wears off to increment the stat

Reply:
Yeah I'm fine with doing that.